python/lsst/sdm/schemas/apdb.yaml

```diff
@@ -1022,6 +565,12 @@ tables:
     datatype: timestamp
     nullable: false
     description: Last time when non-forced DIASource was seen for this object.
+  - name: firstDiaSourceMjdTai
+    "@id": "#DiaObject.firstDiaSourceMjdTai"
+    datatype: double
+    nullable: false
```

Comment:
In case of Cassandra it will not be possible to populate it with realistic data (or any data) for existing records when migrating the schema. Client code will have to handle the situation when returned records still have it as NULL.

Reply:
And I think that BigQuery (PPDB) may have a significant price tag attached if you want to update every row in a large table. @JeremyMcCormick, I think we would want to know how BigQuery handles addition of a new column(s) to a table in two scenarios:
- add a nullable table and do not populate it with any data (I expect this to be fast and cheap),
- add non-nullable column and populate it (I'm not sure that non-nullable even allowed in that sense, but populating every row is likely to be super-expensive).

Another interesting question is how expensive it is to drop a column from a table?

Reply:
Good points all--I will make this nullable.